dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -605,7 +605,7 @@
 
 7342=The specified boolean value [{0}] for setting allow native sql queries is invalid, the value must either be "true" or "false".
 
-7343=Multiple VPD identifiers (tenant discriminator context property) have been specified. Entity [{1}] uses [{0}] and Entity [{3]} uses [{2}]. When using a Multitenant VPD strategy, there can only be one tenant discriminator column per entity and its context property must be consistent across all the Multitenant VPD entities.
+7343=Multiple VPD identifiers (tenant discriminator context property) have been specified. Entity [{1}] uses [{0}] and Entity [{3}] uses [{2}]. When using a Multitenant VPD strategy, there can only be one tenant discriminator column per entity and its context property must be consistent across all the Multitenant VPD entities.
```

Comment:
The "When using," "there can only be" and "per" lower the quality scores. Suggested change for the third sentence:

The Multitenant VPD strategy allows only one tenant discriminator column for each entity and its context property must be consistent across all the Multitenant VPD entities.

Reply:
If I understand you correctly, you don't want to change the message now but will forward the suggested changes to the EclipseLink OS project? If that is your position, then add the ID review not needed label.

Reply:
It's more that I can't change the message in OL.

The message files we have checked into OL are created from the ones from the OS binaries. A validation step checks that the messages are in sync. If I update the `.nlsprops` that we have checked into OL, they will be out of sync with the messages as they appear in the binaries. 

If the suggested message change is minor and not prohibitive to delivering the new binaries, then I will recommend to the OS project so that we can get that change in when the new binaries are created.